### DISCUSSION TITLE
Bot joins then leaves

### DISCUSSION THREAD

**Discussion by Turismo2x** [jagrosh/MusicBot#1057] [General]
so the bot work perfectly fine in my discord server when i add it to my friends server were i have admin and stuff and when i try playing a song it joins and leave everytime can someone help me

**Comment by Turismo2x** [comment]
also is there way to be able to spotify songs and playlists or no

**Reply by Dragondarn1993** [reply]
https://jmusicbot.com/things-that-wont-be-added/

**Comment by hermitoff** [comment]
Hello, I have the same problem. I noticed that when the bot disconnects it makes an error in the console that here: 

I see that Youtube returns the error code 403. Why ?

password is : helpmeplease

error : https://pastebin.com/kfWvdXJe

**Comment by lorivaro** [comment]
You might be experiencing this issue: https://github.com/jagrosh/MusicBot/issues/885
